docs/Manufacturer Design Guidelines.md

```diff
@@ -0,0 +1,345 @@
+
+
+## 1.2 Important Terms and Glossary
+
+| Term | Definition | Reference |
+| :--- | :--- | :--- |
+| ADC | Analog to Digital Converter |
+| BEC | Battery Elimination Circuit | Electronic voltage regulator circuit that provides a specified voltage to a single output power rail |
+| Bidirectional DSHOT | Two-Way DSHOT communication protocol to enable Electronic Speed Controller telemetry to be sent back to the flight controller | https://github.com/betaflight/betaflight/wiki/Bidirectional-DSHOT-and-RPM-Filter |
+| Bit Bang | Bit-banging refers to any instance of utilizing GPIO functionality to digitally create signals in place of dedicated hardware. This can alleviate some specific data buffer timing requirements |
+| DSHOT | Digital Shot Communication Protocol used for Flight Controller to Electronic Speed Controller | https://github.com/betaflight/betaflight/wiki/DSHOT-ESC-Protocol |
+| ESC | Electronic Speed Controller | https://us.aspina-group.com/en/learning-zone/columns/what-is/021/ |
+| FPV | First Person View | This can also refer to the complete avionics-telemetry supplied via a camera & video transmission system |
+| GPIO | General Purpose Input/Output | Reconfigurable digital signal pin that can be selected from an MCU pin |
+| IMU | Inertial Motion Unit | Inertial Navigation hardware systems using combined gyroscope and accelerometer unit intended to provide accurate estimates of angular rates, acceleration, and orientation. These can also incorporate magnetometers, barometers, and GPS signals |
+| I2C | Inter-Integrated Circuit synchronous serial communication protocol used for connecting microcontroller CPU to peripheral devices |
+| MCU | Micro Controller Unit | Single integrated circuit microprocessor with integrated processor, memory, and programmable input/output modules |
+| SPI Bus | Serial Peripheral Interface that enables MCU to interface with ADC, DAC, registers, RAM storage, and GPIO | https://www.analog.com/en/analog-dialogue/articles/introduction-to-spi-interface.html |
+| PCB | Printed Circuit Board | May also refer to a populated PCB with components
+| PDB | Power Distribution Board | A PCB intended to provide power distribution to electronic speed controller, flight controller, and other peripheral elements |
+| VDD | Voltage - Operating Voltage for a particular chip |
+| VDD_IO | Voltage range permissible as I/O Supply voltage (Typically 3.3V or less) |
+
+
+# 2 Engaging with the Betaflight Development Team
+
+## 2.1 Hardware Approval Process - Example
+
+- Manufacturer initiates contact with Betaflight developer(s) at hardware@betaflight.com
```

Comment:
Is this address active and forwarded to the right team members ?

Reply:
Let me know who should get emails :)